docs/manage/fleet/_index.md

```diff
@@ -56,30 +57,52 @@ Role Based Access Control (RBAC) ia a way to enforce security in the [Viam app](
 
 A user can have one or more roles, granting the user the respective permissions of each role.
 
-## Configuration
+## Collaborate on your robots
+
+Viam is built in a way that allows you to change configurations, deploy packages, check logs, and control your robots both when you are close to your robot, as well as remotely.
+
+Robot [configuration](robots/#configuration) and Robot [code](#control-with-code) is intentially kept separate, allowing you to keep track of versioning and debug issues separately.
 
-When a robot or a {{< glossary_tooltip term_id="part" text="robot part" >}} that is managed with the Viam app first comes online, it requests its configuration from the [Viam app](https://app.viam.com).
-Once the robot has a configuration, it caches it locally and can use the configuration for up to 60 days.
-The robot checks for new configurations every 15 seconds and changes its configuration automatically when a new configuration is available.
+### Configuration
 
-{{< alert title="Tip" color="tip" >}}
-If you are managing a large fleet, you can use {{< glossary_tooltip term_id="fragment" text="fragments" >}} when [configuring your robot](../configuration).
+Everyone who has access to the location the robot is in, can change the robot's configuration.
+When you or your collaborators change a robot's configuration, the robot will automatically reconfigure itself within 15 seconds.
+You can see configuration changes from yourself or your collaborators on the [History tab](robots/#history).
+You can also revert to an earlier configuration from the History tab.
+
+{{< alert title="Note" color="note" >}}
+For some configuration aspects you may require physical access to the robot so you can see how components are connected.
 {{< /alert >}}
 
-## Package Deployment
+### Package Deployment
 
 _Coming soon._
 
-Deploy control logic, [modular resources](/program/extend/modular-resources/), sidecar [processes](../configuration/#processes), or [machine learning models](../../services/ml/) to your fleet of robots without manually copying files by uploading it to Viam's cloud and deploying it to your fleet.
+You and your collaborators can deploy control logic, [modular resources](/program/extend/modular-resources/), sidecar [processes](../configuration/#processes), or [machine learning models](../../services/ml/) to your fleet of robots without manually copying files by uploading it to Viam's cloud and deploying it to your fleet.
+
+### Remote control
+
+Everyone who has access to the location the robot is in, can remotely control the robot using the [**control** tab](robots/#control).
```

Comment:
This "Everyone who has access to the location the robot is in" phrase being repeated makes me have de ja vu. Maybe after the first time it can be shortened to "Everyone"? Or
```suggestion
Everyone who has access to the robot can remotely control it using the [**control** tab](robots/#control).
```

Reply:
Then it's only clear if you read everything in order which we can't assume